ee/codegen/src/context/workflow-context/workflow-context.ts

```diff
@@ -109,6 +118,29 @@ export class WorkflowContext {
     this.workflowRawEdges = workflowRawEdges;
   }
 
+  /* Create a new workflow context for a nested workflow from its parent */
```

Comment:
Introduces a helper for creating a child workflow context and moves/adapts a similar helper here from below.